public/js/reader.js

```diff
@@ -140,11 +140,21 @@ Reader.initializeAll = function () {
             // Regex look in tags for artist
             const artist = data.tags.match(/artist:([^,]+)(?:,|$)/i);
             if (artist) {
-                title = `${title} by ${artist[1]}`;
+                const artistName = artist[1];
+                const artistSearchUrl = `/?sort=0&q=artist%3A${encodeURIComponent(artistName)}%24&`;
+                const link = $('<a></a>')
+                    .attr('href', artistSearchUrl)
+                    .attr('style', 'color: inherit; text-decoration: underline;')
```

Comment:
I'm not sure this warrants a style override compared to the normal look of links within the app.